src/StandupMeetingReminder/Handler.php

```diff
@@ -377,11 +471,24 @@ private function displayModalForm(string $teamId, ?string $channelId, string $tr
 
                 foreach ($forecastAccounts as $forecastAccount) {
                     $this->forecastDataSelector->setForecastAccount($forecastAccount);
+                    $clients = $this->forecastDataSelector->getClients(true);
                     $projects = $this->forecastDataSelector->getProjects(true);
 
+                    foreach ($clients as $client) {
+                        if (\in_array((string) $client->getId(), $standupMeetingReminder->getForecastClients(), true)) {
```

Comment:
This is weird, it's said the array contains `int`, but here you cast in `string`. more over, there is `strict=true` as third arg